frontend/ui/elements/screensaver_menu.lua

```diff
@@ -69,7 +69,7 @@ return {
             {
                 text = _("Postpone screen update after wake-up"),
                 sub_item_table = {
-                    genMenuItem(_("No delay"), "screensaver_delay", "disable"),
+                    genMenuItem(_("Never"), "screensaver_delay", "disable"),
```

Comment:
I think we discussed it at the time you made the PR.
"Never" feels a bit ambiguous, it's unclear whther the screen will never update - or if it will never postpone the update. "No delay" was more obvious.